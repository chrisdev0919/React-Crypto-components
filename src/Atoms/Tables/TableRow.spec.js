import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import TableRow from './TableRow'

describe('TableRow component', () => {
  it('renders correctly', () => {
    const component = shallow(<TableRow>MyTableRow</TableRow>)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
