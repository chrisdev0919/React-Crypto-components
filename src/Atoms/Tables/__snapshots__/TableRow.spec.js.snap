// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableRow component renders correctly 1`] = `
<styled.div>
  MyTableRow
</styled.div>
`;
